packages/javalib.2.3.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.1.1"
+name: "javalib"
+maintainer: "sawja@inria.fr"
+version: "2.3.1"
```

Comment:
`opam-version` specifies the minimum version of opam required to read the metadata
`version` specifies the version of the library but this information is duplicated from the path in the opam-repository.